cttportfolio.html

```diff
@@ -141,7 +142,7 @@ <h4>Contact me</h4>
 			<div class="contactmetext">
 				<div class="contact_div background_color">
 					<p>Want to get in touch?</p>
-					<h6>E-mail: christianturner@hotmail.co.uk</h6>
+					<h6>E-mail: christianturner @hotmail.co.uk</h6>
```

Comment:
Email addresses don't have spaces.

Reply:
i added the space so that when its seens on the phone the font isn't tiny.

Reply:
I tried break-word but it would make it look worse